src/views/homepage/homepage.module.css

```diff
@@ -9,7 +9,7 @@
 
 .background {
 	background-color: var(--token-color-foreground-strong);
-	background-image: url('./img/background.svg');
+	background-image: url('./img/background.png');
```

Comment:
🗒️ We had to change this to a `PNG` because the `SVG` wasn't getting exported correctly.

Reply:
I noticed this image is pretty big. I was able to get 14% savings by running it through ImagOptim. There's probably other optimizations that could be made to compress it further.

Reply:
Agreed, we need to make this dude smaller or try as best we can with the svg to get it working? The svg is probably a fraction of the size.

Reply:
Got some savings with Image Optim on my machine!

Reply:
Imo I think we should aim to get this closer to or below 1MB, 1.8Mb definitely feels too big, especially when thinking of mobile. Perhaps we can play with exporting at a slightly smaller size? And yeah with imageOptim I saw 27% savings on this one so that could be enough perhaps?